### PR TITLE
transcribe: return all language probabilities if requested

### PR DESCRIPTION
**Pull request by ozancaglayan**
This change adds a `all_language_probs` field to `TranscriptionInfo` so that when language detection is done, we can recover the full probabilities from the model in addition to the top one.

### CODE REVIEW

File: faster_whisper/transcribe.py [hunk 1]
```diff
@@ -150,6 +151,7 @@ def transcribe(
         self,
         audio: Union[str, BinaryIO, np.ndarray],
         language: Optional[str] = None,
+        return_all_language_probs: bool = False,
```
Comment (guillaumekln) [inline faster_whisper/transcribe.py:154]: I'm OK to always return the language probabilities when they are available. So I suggest to simply remove this argument.

**Review by guillaumekln — CHANGES_REQUESTED** [review]
I think the new argument is not needed.

File: faster_whisper/transcribe.py (hunk 1 above)
Comment (ozancaglayan) [reply faster_whisper/transcribe.py:154]: Sure, thanks! To confirm, we'll still keep the top language name and prob for backward compatibility right?

Comment (guillaumekln) [reply faster_whisper/transcribe.py:154]: Yes, keep the existing properties and add `all_language_probs` (which can still be `None` if the language detection did not run).

**Comment by ozancaglayan** [comment]
Test fails because Huggingface hub is down since some hours.

**Pull request merged**